CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+# 0.10.0
+
+## Features
+
+- **M3Query**: Add multi-zonal capability to remote reads (#1687)
```

Comment:
@arnikola Same comment about docs

Reply:
Can we make this `Add multi-zone and multi-region configuration for coordinator (#1687)`?